skimage/measure/fit.py

```diff
@@ -800,13 +806,22 @@ def ransac(data, model_class, min_samples, residual_threshold,
     # number of samples
     num_samples = data[0].shape[0]
 
+    if init_inliers is not None and len(init_inliers) != num_samples:
+        raise ValueError("`number init samples %i are les then data %i "
```

Comment:
Typo (les -> less). Also, conceivably, the number could be more, not less!

As we are discussing data types in the mailing list, I'm becoming more aware that error messages should be more specific about the cause of the error *and* how to fix it. An example of what this message could be:

"RANSAC received a vector of initial inliers (length %i) that didn't match the number of samples (%i). The vector of initial inliers should have the same length as the number of samples and contain only True (this sample is an initial inlier) and False (this one isn't) values."

@stefanv care to comment? This would be a significant policy change but I think it would be very positive.